src/menu/charprofile.c

```diff
@@ -154,23 +154,6 @@ static const CharacterProfile profiles[NUM_PROFILES] = {
 	}
 };
 
-
-static void update_char_draw_order(MenuData *m) {
-	CharProfileContext *ctx = m->context;
-
-	for(int i = 0; i < NUM_PROFILES; ++i) {
-		if(ctx->char_draw_order[i] == m->cursor) {
-			while(i) {
-				ctx->char_draw_order[i] = ctx->char_draw_order[i - 1];
-				ctx->char_draw_order[i - 1] = m->cursor;
-				--i;
-			}
-
-			break;
-		}
-	}
-}
-
```

Comment:
Should remove the draw order stuff from the context struct as well, if you're removing this.